docs/src/components/LoadingIndicator/Basic.tsx

```diff
@@ -1,39 +1,30 @@
-import {LoadingIndicator, ThemeProvider, ThemeType, useTheme} from 'dooboo-ui';
+import {LoadingIndicator, ThemeProvider, ThemeType} from 'dooboo-ui';
 
 import type {FC} from 'react';
-import {View} from 'react-native';
+import styled from '@emotion/native';
 
-export const Component: FC = () => {
-  const {theme} = useTheme();
+const View = styled.View`
+  height: 60;
+  background-color: ${({theme}) => theme.background};
+  align-self: 'stretch';
+  justify-content: 'center';
+`;
```

Comment:
Please name it something other than `View`. Because it can be misunderstood.
I suggest something like `Container` or `StyledView`.